std_msgs/Header header
int32 num_agent
int32 num_waypoint
float64 scale
float64 minx
float64 miny
float64 maxx
float64 maxy
geometry_msgs/PoseArray[] goals